tests/test.py

```diff
@@ -1035,44 +1038,36 @@ def run_cli(*, image: str):
     LOG.info("%s passed %d integration tests", image, num_tests_ran)
 
 
-def run_security(*, image: str):
+def run_security(*, image: str, variant: str):
     """Run the security tests"""
-    tag = image.split(":")[-1]
-    file_name = f"{tag}.tar"
-    file_path = utils.write_docker_image(image=image, file_name=file_name)
-    file_dir = file_path.parents[0]
+    artifact_labels = utils.get_artifact_labels(variant=variant)
+    label = artifact_labels[0]
+    sbom_file = Path(f"sbom.{label}.json")
 
-    working_dir = "/tmp/"
-    volumes = {file_dir: {"bind": working_dir, "mode": "ro"}}
-
-    num_tests_ran = 0
-    scanner = constants.CONTAINER_SECURITY_SCANNER
-
-    # Provide debug information about unknown, low, and medium severity
-    # findings
-    command = (
-        "--quiet image --exit-code 0 --severity "
-        + ",".join(constants.INFORMATIONAL_VULNS)
-        + " --format json --light --input "
-        + working_dir
-        + file_name
-    )
-    utils.opinionated_docker_run(
-        image=scanner, command=command, volumes=volumes, expected_exit=0
-    )
-    num_tests_ran += 1
+    if not sbom_file:
+        LOG.error(
+            f"{sbom_file} was not found; security scans require an SBOM. Please run `pipenv run invoke sbom`"
+        )
 
-    # Ensure no critical vulnerabilities exist in the image
-    command = (
-        "--quiet image --exit-code 1 --severity "
-        + ",".join(constants.UNACCEPTABLE_VULNS)
-        + " --format json --light --input "
-        + working_dir
-        + file_name
-    )
-    utils.opinionated_docker_run(
-        image=scanner, command=command, volumes=volumes, expected_exit=0
-    )
-    num_tests_ran += 1
+    # Run a vulnerability scan on the provided SBOM (derived from variant)
+    try:
+        LOG.info(f"Running a vulnerability scan on {sbom_file}...")
+        subprocess.run(
+            [
+                "grype",
```

Comment:
@callmestu notably this does not fail on any level of finding.  I broke this out into a [separate commit](https://github.com/SeisoLLC/easy_infra/pull/143/commits/faa83f87c0d7061afae3d9d1043fa3bcc35a6141#diff-002b17638e4a840ab803fb150dddd393a637a14314ed5c395a048192850b2603) in the hopes that it would be easier to review

Reply:
Reviewed changes.